js_modules/dagit/packages/core/src/nav/InstanceWarningIcon.test.tsx

```diff
@@ -7,12 +7,6 @@ import {TestProvider} from '../testing/TestProvider';
 import {InstanceWarningIcon} from './InstanceWarningIcon';
 
 describe('InstanceWarningIcon', () => {
-  const defaultMocks = {
-    DaemonHealth: () => ({
-      allDaemonStatuses: () => [...new Array(3)],
```

Comment:
I removed this base mocking because I think this behavior (three daemon statuses in random states) isn't a helpful default for testing.